README.md

```diff
@@ -2,31 +2,200 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Introduce database management systems concepts
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Databases often contain massive amounts of data and support thousands
+of simultaneous users, and thus must be managed with sophisticated software tools.
 
-## Topic 1
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+## Relational Database Management Systems
 
-## Topic 2
+- **Data** is information that can be represented in many formats
+  including numeric, textual, visual, or audio.
+- A **database** is a collection of data in a structured format.
+- A **database management systems** or **DBMS** is software that reads and writes data
+  and often provides additional features including security, performance, transactions, multi-user access, and recovery. 
+- A **relational database management system** or **RDBMS** stores data in tables, columns and rows.
+  - A table holds information about objects of a similar type.
+  - Each column within a table has a unique name.
+  - All values within a column have the same type and are atomic.
+  - Each row within a table represents a unique object and must have a unique identifier, called a **primary key**.
+  - Rows among multiple tables can be related using **foreign keys**, in which one or more columns in
+    one table store the values of the primary key of another table.
+- A **query language** is a specialized programming language designed for database systems.
+  - **Structured Query Language** or **SQL** is a query language designed for relational databases.
+- **PostgreSQL** is a popular open source RDBMS.  Other RDMBS products include Oracle, MySQL, SQLite, and SQL Server.
+
+## Database Structure
+
+Relational databases store data in table that consists of
```

Comment:
Small typo: "Relational databases store data in table that..."

Should be "Relational databases store data in a table that..."

Reply:
fixed.